### PR TITLE
Add readme.md

### PR DESCRIPTION
**Pull request by bullet133790**

### CODE REVIEW

File: src/task1.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import java.lang.reflect.Parameter;
+import java.util.Scanner;
+
+public class task1 {
```
Comment (tallinn-learning-ilja) [inline src/task1.java:4]: First of all, all class names should begin with the capital letter. 
Another thing, in the homework description it was stated that everything HAS to be in one class.

File: src/task2.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import java.util.Scanner;
+
+public class task2 {
```
Comment (tallinn-learning-ilja) [inline src/task2.java:3]: Again, capital letter

File: src/task3.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+
+public class task3 {
```
Comment (tallinn-learning-ilja) [inline src/task3.java:3]: same

File: src/task4.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import java.util.Scanner;
+
+public class task4 {
```
Comment (tallinn-learning-ilja) [inline src/task4.java:3]: Same

File: src/task4.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    public static void main(String[] args) {
```
Comment (tallinn-learning-ilja) [inline src/task4.java:4]: Where is your method? Main method is only used for executing code

**Review by tallinn-learning-ilja — CHANGES_REQUESTED** [review]
Please move all methods under ONE class. Name class properly, something like `Homework` or `Exercise`

File: src/task1.java (hunk 1 above)
Comment (bullet133790) [reply src/task1.java:4]: I corrected everything you said. and pushed again

File: src/Homework3.java [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        return word.charAt(0);
+    }
+//3
+    public static char getLastChar(String word3) {
+        return word3.charAt(word3.length() - 1);
+    }
+
+//4
+    public static String swapFirstAndLastCharacters (String word4){
+            return word4.charAt(word4.length() - 1) + word4.substring(1, word4.length() - 1) + word4.charAt(0);
```
Comment (tallinn-learning-ilja) [inline src/Homework3.java]: Please make those methods static

File: src/Homework3.java [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+//1
+        public static int absoluteDifference(int first, int second) {
+            return Math.abs(first - second);
+//2
+        }
+    public static char getFirstChar(String word) {
+        return word.charAt(0);
+    }
+//3
+    public static char getLastChar(String word3) {
```
Comment (tallinn-learning-ilja) [inline src/Homework3.java]: it can be just `String word` instead of `word3`

File: src/Homework3.java [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+        return word3.charAt(word3.length() - 1);
+    }
+
+//4
+    public static String swapFirstAndLastCharacters (String word4){
```
Comment (tallinn-learning-ilja) [inline src/Homework3.java]: this one can also be `word`

**Review by tallinn-learning-ilja — CHANGES_REQUESTED** [review]
Some more minor things and then it's okay

File: src/Homework3.java (hunk 6 above)
Comment (bullet133790) [reply src/Homework3.java]: can you explain me please what you mean?? because they are all public static.

Comment (tallinn-learning-ilja) [reply src/Homework3.java]: Sorry,  I meant non-static

Comment (bullet133790) [reply src/Homework3.java]: Done

File: src/Homework3.java [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+            //1
+            Scanner scanner = new Scanner(System.in);
+            System.out.println("Enter the first number");
+            int first = scanner.nextInt();
+            System.out.println("Enter the second number");
+            int second = scanner.nextInt();
+            int result = homework.absoluteDifference(first,second);
+            System.out.println("Result " + result );
+            //2
+            Scanner scanner2 = new Scanner(System.in);
```
Comment (tallinn-learning-ilja) [inline src/Homework3.java:16]: You don't need to create multiple scanners

File: src/Homework3.java [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+            System.out.println("Type any word");
+            String word = scanner2.nextLine();
+            char firstChar = homework.getFirstChar(word);
+            System.out.println("Result = " + firstChar);
+            //3
+            Scanner scanner3 = new Scanner(System.in);
```
Comment (tallinn-learning-ilja) [inline src/Homework3.java:22]: Here as well, one scanner is enough

**Review by tallinn-learning-ilja — APPROVED** [review]
Good, small thing for your information

**Pull request merged**